scripts/ubuntu_build_setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+function install_dependencies {
+  echo "Install dependencies"
+  apt update
+  apt install -yq git build-essential python3.8 python3-pip pkg-config mesa-common-dev \
+     libglu1-mesa-dev wget libpcsclite-dev libpcre3-dev libssl-dev libpulse-mainloop-glib0 \
+     libxkbcommon-x11-dev extra-cmake-modules cmake
+
+}
+
+function install_qt {
+  echo "Install QT"
+  pip install -U pip
+  pip install aqtinstall
+  mkdir /opt/qt
+  cd /opt/qt
+  aqt install-qt linux desktop 5.14.2 gcc_64 -m qtwebengine qtlottie
```

Comment:
`aqtinstall` has a `--outputdir` flag:
https://aqtinstall.readthedocs.io/en/latest/getting_started.html#changing-the-output-directory